modules/plugins/youtube.go

```diff
@@ -105,12 +118,34 @@ func (yt *YouTube) actionFinish(args []string, in *discordgo.Message, out **disc
 	return nil
 }
 
+// DEBUG PURPOSE
+func (yt *YouTube) actionQuota(args []string, in *discordgo.Message, out **discordgo.MessageSend) youtubeAction {
+	if helpers.IsBotAdmin(in.Author.ID) == false {
+		*out = yt.newMsg("botadmin.no_permission")
+		return yt.actionFinish
+	}
+
+	c := yt.quota.GetContent()
+
+	msg := fmt.Sprintf("left time to reset: `%d`, left quota: `%d`, channel count: `%d`, time interval: `%d`",
+		c.ResetTime-time.Now().Unix(),
```

Comment:
I think using something like `time.Unix(c.ResetTime, 0).Format(time.ANSIC)`, and saying "next reset:" in the message would be more useful

Reply:
changed